its/projects/MultipleProjects/opencover.xml

```diff
@@ -0,0 +1,1721 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should we add a small comment to document the exact command used to generate the file (in case we need to update the code and generate this again)?

Reply:
yes, I am adding a README in the next PR with visual studio coverage

Reply:
Indeed, having it in a separate file sounds better.